### PR TITLE
Extend root owners to help CI/CD PRs

### PR DESCRIPTION
**Pull request by kimwnasptd**
Right now in order to merge a PR that modifies `kubeflow/kubeflow`'s [prow_config.yaml](https://github.com/kubeflow/kubeflow/blob/0c4a738625f52092f66ec462e800ddd2837e0f6d/prow_config.yaml) file, i.e https://github.com/kubeflow/kubeflow/pull/5600, we need to ping someone from the existing OWNERS file.

In order to reduce the friction of expecting users that are not tightly coupled with the CI/CD effort #5482  of this repo, like @Bobgy, we should add @PatrickXYS and myself in the root OWNERS file. 

This will allow us to move faster with our PRs wrt to the CI/CD of the `kubeflow/kubeflow` repo, without requiring cycles from developers that are not working on this part.

/cc @Bobgy

### CODE REVIEW

**Comment by PatrickXYS** [comment]
Thanks lgtm

If you want to bypass the PR without failing tests, you can comment out the prow_config.yaml in this PR.

We can restore prow_config.yaml in other PR afterwards

**Comment by kimwnasptd** [comment]
Good point @PatrickXYS, I commented out the unnecessary workflow. The tests obviously pass now since no tests are run

**Review by PatrickXYS — APPROVED** [review]
/lgtm
/approve

**Comment by Bobgy** [comment]
I'm asking for internal approval, because kubeflow/kubeflow an important public facing repo.

Some ideas for workarounds:
* @PatrickXYS is it possible to configure location of prow_config.json, so it's not in the root folder
* @kimwnasptd what would your opinion be if we can create a kubeflow/notebooks repo and move all notebooks development there? (I'm not sure whether it can be approved, but just want to get your thoughts first too.)

In the meanwhile I can try to help you approve the initial PRs.

**Comment by DavidSpek** [comment]
@Bobgy With a `kubeflow/notebooks` repo, this repo would be almost empty and mainly server as a public facing repo where people can aks general questions and get linked through to the repositories of each WG/component. Personally, I don't think this would be a terrible move. I think this would also tackle https://github.com/kubeflow/community/issues/344 and https://github.com/kubeflow/kubeflow/issues/5418. 
/cc @thesuperzapper

**Comment by kimwnasptd** [comment]
Thank you very much @Bobgy!

Since we are getting really close to the 1.3 release I'd prefer to pick a solution that will unblock us right now, so we get our infra in order. I believe making the `prow_config.yaml` file's position configurable, at least for this repo, should not be to difficult to patch right, now but lets first hear @PatrickXYS's input on this.

So if we can get unblocked with just a small change in the `kubeflow/testing` repo's py-script that parses the `prow_config.yaml` files then lets go with this for now.

> @kimwnasptd what would your opinion be if we can create a kubeflow/notebooks repo and move all notebooks development there? (I'm not sure whether it can be approved, but just want to get your thoughts first too.)

I think it makes sense to discuss about a `kubeflow/notebooks` repo, but let's do it after 1.3 is out and we have our processes in order first.

**Comment by PatrickXYS** [comment]
> So if we can get unblocked with just a small change in the kubeflow/testing repo's py-script that parses the prow_config.yaml files then lets go with this for now.

This will required ad-hoc changes in kubeflow/testing, I would suggest us avoid doing this since other kubeflow repos are following the same pattern: root `prow_config.yaml`.

I'm a bit confused if we have already granted notebook-WG with kubeflow/kubeflow write access, changing kubeflow/kubeflow root OWNERS should be desired as we assume. 

If we can not 

1) Extend root owners
2) Make ad-hoc changes in kubeflow/testing

WG-notebooks, feel free to merge PR manually if changed files related to root folder

**Comment by Bobgy** [comment]
> what would your opinion be if we can create a kubeflow/notebooks repo and move all notebooks development there

OK, I can confirm we can go this path after current release. Let's use a different issue to track that.

I'm OK with you merging manually things you want to change using repo write permission. I'd only suggest to ask broader review if you want to make changes to root README (I think you know this).

**Comment by kimwnasptd** [comment]
@Bobgy thank you very much for your effort!

One last question I'd like to clarify to make sure we are in sync for what the next steps should be:
> I'm OK with you merging manually things you want to change using repo write permission.

Are you referring to the `kubeflow/kubeflow` repo? If yes then we can't merge things manually since we don't have write permissions for root level files, like `prow_config.yaml`. 

So the plan to unblock right now would be for you to help review our changes in the `prow_config.yaml` file, while we set up our CI/CD? Ofc we don't expect you to be deep diving into the changes, @PatrickXYS can help with the reviews. Just want to to make it clear how we should move forward right now.

> I'd only suggest to ask broader review if you want to make changes to root README (I think you know this).

I think this goes without saying for any repo/project/collaborative-effort. So definitely ACK!

**Comment by PatrickXYS** [comment]
> Are you referring to the kubeflow/kubeflow repo? If yes then we can't merge things manually since we don't have write permissions for root level files, like prow_config.yaml.

Emmm seems like you're not granted with write access as other WGs did before, https://github.com/kubeflow/internal-acls/blob/master/github-orgs/kubeflow/org.yaml#L1027-L1035

I think for current work-around, we can have @Bobgy  help approve after we reviewed and lgtm, I think this might makes everyone happy in current stage.

**Pull request closed without merging**
